pkgs/development/python-modules/catboost/default.nix

```diff
@@ -55,4 +55,9 @@ buildPythonPackage rec {
   doCheck = false;
 
   pythonImportsCheck = [ "catboost" ];
+
+  meta = catboost.meta // {
+    # Incompatible with numpy >= 2.0
+    broken = true;
```

Comment:
Shall we link to the tracking issue? https://github.com/catboost/catboost/issues/2671